src/assets.php

```diff
@@ -39,15 +40,24 @@ function action_wp_enqueue_scripts(): void {
  */
 function action_newsletters_enqueue_styles(): void {
 	$blocks = [
-		'header',
+		'button',
+		'divider',
 		'footer',
+		'header',
+		'heading',
+		'list',
+		'paragraph',
 		'post',
-		'two-up-post',
 		'section',
+		'two-up-post',
```

Comment:
this can actually stay the same (feel free to alphabetize), because lines 73-84 will handle the core blocks.

Reply:
done!